
Scenario: Test cases that pass aren't retried # samples/retry/retry.feature:8
  Given a step that always passes             # samples/retry/retry.feature.ts:3

Scenario: Test cases that fail are retried if within the --retry limit # samples/retry/retry.feature:11
  Given a step that passes the second time                             # samples/retry/retry.feature.ts:8
      Exception in step

Scenario: Test cases that fail are retried if within the --retry limit # samples/retry/retry.feature:11
  Given a step that passes the second time                             # samples/retry/retry.feature.ts:8

Scenario: Test cases that fail will continue to retry up to the --retry limit # samples/retry/retry.feature:14
  Given a step that passes the third time                                     # samples/retry/retry.feature.ts:16
      Exception in step

Scenario: Test cases that fail will continue to retry up to the --retry limit # samples/retry/retry.feature:14
  Given a step that passes the third time                                     # samples/retry/retry.feature.ts:16
      Exception in step

Scenario: Test cases that fail will continue to retry up to the --retry limit # samples/retry/retry.feature:14
  Given a step that passes the third time                                     # samples/retry/retry.feature.ts:16

Scenario: Test cases won't retry after failing more than the --retry limit # samples/retry/retry.feature:17
  Given a step that always fails                                           # samples/retry/retry.feature.ts:23
      Exception in step

Scenario: Test cases won't retry after failing more than the --retry limit # samples/retry/retry.feature:17
  Given a step that always fails                                           # samples/retry/retry.feature.ts:23
      Exception in step

Scenario: Test cases won't retry after failing more than the --retry limit # samples/retry/retry.feature:17
  Given a step that always fails                                           # samples/retry/retry.feature.ts:23
      Exception in step

Scenario: Test cases won't retry when the status is UNDEFINED # samples/retry/retry.feature:20
  Given a non-existent step
